src/views/List/List.jsx

```diff
@@ -25,9 +25,15 @@ export default function List({ token }) {
         data.id = doc.id;
         snapshotDocs.push(data);
       });
-      setData(snapshotDocs);
+      const sortedName = snapshotDocs.sort((a, b) => {
+        return a.name.localeCompare(b.name);
```

Comment:
I learned something new 🎉

localeCompare is a way to sort alphabetically with ignoring letter casing https://bobbyhadz.com/blog/javascript-sort-ignore-case